spec/features/brexit_checker/accounts_spec.rb

```diff
@@ -0,0 +1,263 @@
+require "spec_helper"
+
+RSpec.feature "Brexit Checker accounts", type: :feature do
+  context "without accounts enabled" do
+    let(:mock_results) { %w[nationality-eu] }
+
+    context "/transition-check/questions" do
+      it "does not show the login state" do
+        given_im_on_a_question_page
+        expect(page).to_not have_content("Logged in.")
+        expect(page).to_not have_content("Not logged in.")
+      end
+    end
+
+    context "/transition-check/results" do
+      it "does not show the login state" do
+        given_im_on_a_question_page
+        expect(page).to_not have_content("Logged in.")
+        expect(page).to_not have_content("Not logged in.")
+      end
+    end
+
+    context "/transition-check/saved-results" do
+      it "returns a 404" do
+        given_im_on_the_saved_results_page
+        expect(page.status_code).to eq(404)
+      end
+    end
+
+    def given_im_on_a_question_page
+      visit transition_checker_questions_path
+    end
+
+    def given_im_on_the_results_page
+      visit transition_checker_results_path(c: mock_results)
+    end
+
+    def given_im_on_the_saved_results_page
+      visit transition_checker_saved_results_path
+    end
+  end
+
+  context "with accounts enabled" do
+    let(:attribute_service_url) { "http://attribute-service" }
+
+    before do
+      ENV["OIDC_CLIENT_ID"] = "id"
+      ENV["OIDC_CLIENT_SECRET"] = "secret"
+
+      allow(Services).to receive(:accounts_api).and_return(Plek.find("account-manager"))
```

Comment:
Isn't this just returning what `Services.acounts_api` will already produce?

Reply:
Good spot!  I had reordered and rebased some commits but didn't spot this wouldn't be needed anymore.